src/apps/bridge/aanderaaController.cpp

```diff
@@ -242,6 +254,12 @@ static void runController(void *param) {
               if (agg.temp_mean_deg_c < TEMP_SAMPLE_MEMBER_MIN || agg.temp_mean_deg_c > TEMP_SAMPLE_MEMBER_MAX) {
                 agg.temp_mean_deg_c = NAN;
               }
+              if (agg.abs_tilt_mean_rad < DIRECTION_SAMPLE_MEMBER_MIN || agg.abs_tilt_mean_rad > DIRECTION_SAMPLE_MEMBER_MAX) {
+                agg.abs_tilt_mean_rad = NAN;
+              }
+              if (agg.std_tilt_mean_rad < DIRECTION_SAMPLE_MEMBER_MIN || agg.std_tilt_mean_rad > DIRECTION_SAMPLE_MEMBER_MAX) {
+                agg.std_tilt_mean_rad = NAN;
+              }
```

Comment:
Max tilt is actually different — π/2 (that is, `M_PI_2` from math.h) instead of 2π.